manifests/Makefile

```
@@ -1,19 +1,31 @@
 TARGET_CRD_FILES := $(patsubst upstream/%/crd,crd/%.yaml,$(wildcard upstream/**/crd))
 TARGET_MANIFEST_FILES := $(patsubst upstream/%/base,%.yaml,$(wildcard upstream/**/base))
+TARGET_TEMPLATE_FILES := $(patsubst upstream/%/template,template/%.yaml.raw,$(wildcard upstream/**/template))
 
-build: build-crds build-base
+default: build
+
+clean: clean-crds clean-manifests clean-templates
+clean-crds:
+	rm $(TARGET_CRD_FILES) || true
+clean-manifests:
+	rm $(TARGET_MANIFEST_FILES) || true
+clean-templates:
+	rm $(TARGET_TEMPLATE_FILES) || true
+
+build: build-crds build-base build-templates
 build-crds: $(TARGET_CRD_FILES)
 build-base: $(TARGET_MANIFEST_FILES)
+build-templates: $(TARGET_TEMPLATE_FILES)
 
 define BUILD_HELM_CHART
 	@if [ -d "upstream/$*/helm" ]; then \
 		echo "Generate templates for helm chart defined in upstream/$*/helm..." && \
 		cd upstream/$*/helm && \
 		$(HELM) template $$($(YQ) r chart.yaml chart_name) --values values.yaml --output-dir ./generated_templates -n $$($(YQ) r chart.yaml namespace) --version $$($(YQ) r chart.yaml version) --repo=$$($(YQ) r chart.yaml repo) --include-crds && \
 		generated_yaml_files=$$(find generated_templates -iname '*.yaml') && \
-		echo $$generated_yaml_files | xargs -n1 -I % $(YQ) d -d'*' -i % '**."app.kubernetes.io/managed-by"' && \
-		echo $$generated_yaml_files | xargs -n1 -I % $(YQ) d -d'*' -i % '**."app.kubernetes.io/instance"' && \
-		echo $$generated_yaml_files | xargs -n1 -I % $(YQ) d -d'*' -i % '**."helm.sh/chart"'; \
+		echo $$generated_yaml_files | sed 's/ /\n/g' | xargs -n 1 -I % $(YQ) d -d '*' -i % '**."app.kubernetes.io/managed-by"' && \
+		echo $$generated_yaml_files | sed 's/ /\n/g' | xargs -n 1 -I % $(YQ) d -d '*' -i % '**."app.kubernetes.io/instance"' && \
+		echo $$generated_yaml_files | sed 's/ /\n/g' | xargs -n 1 -I % $(YQ) d -d '*' -i % '**."helm.sh/chart"'; \
```

Are you on Linux? Looks like this won't work for MacOS. I got error
```
Error: stat %: no such file or directory
Usage:
  yq delete [yaml_file] [path_expression] [flags]
```

I'm finding a working version for both Mac and Linux...

---

This should work for both Unix and Linux. Tested on my Mac and an Ubuntu VM.

```suggestion
		printf "%s" "$$generated_yaml_files" | xargs -n 1 -I % $(YQ) d -d '*' -i % '**."app.kubernetes.io/managed-by"' && \
		printf "%s" "$$generated_yaml_files" | xargs -n 1 -I % $(YQ) d -d '*' -i % '**."app.kubernetes.io/instance"' && \
		printf "%s" "$$generated_yaml_files" | xargs -n 1 -I % $(YQ) d -d '*' -i % '**."helm.sh/chart"'; \
```